client/src/components/ReroutToLogin.js

```diff
@@ -0,0 +1,8 @@
+import {Redirect} from "react-router-dom";
+import React from "react";
+
+export const ReroutToLogin = (props) => {
+    return (
+        <Redirect to="/sign-in" setUser={props.setUser}/>
```

Comment:
Why does the Redirect component need the setUser attribute if it is only serving to redirect back to the main page? The user needs to re-login anyways. Maybe I'm not understanding how it works though.

Reply:
Agreed, no user is set here and we want to force them to log in. Additionally, you aren't rendering a component, you are using the route, and the component will get rendered from routes.js so I honestly am not sure it would work anyways